src/InputWithLabel.jsx

```diff
@@ -0,0 +1,24 @@
+import { useRef } from "react";
+
+function InputWithLabel({ todoTitle, handleTitleChange }, props) {
+  const inputRef = useRef(null);
+
+  const focus = () => inputRef.current.focus();
+
+  return (
+    <>
+      <label htmlFor="todoTitle">{props.children}</label>
+      <input
+        id="todoTitle"
+        name={props.children}
+        value={todoTitle}
+        onChange={handleTitleChange}
+        autoFocus
+        ref={inputRef}
+      ></input>
+      <button onClick={focus}>Add</button>
```

Comment:
I would move this `button` outside the component back into `AddTodoForm`. Logically, this component should only contain the label and input.

To manage the focus setting, per the assignment instructions try to incorporate a `useEffect` on this component to set focus on each render (using a useEffect without a dependency array)